src/styles/blocks/card.scss

```diff
@@ -0,0 +1,66 @@
+.card {
+  background-color: $background-color-white;
+  border: 1px solid $border-color;
+  width: 200px;
+  height: 408px;
```

Comment:
There is no need to write a fixed height, it will be better if it depends on the content
```suggestion
```